lib/directory-tree.js

```diff
@@ -25,17 +25,26 @@ function directoryTree (path, options, onEachFile) {
 	const item = { path, name };
 	let stats;
 
+    let depth = 1;
+    if(!options){
```

Comment:
Please include a space after `if` and before `{`, such that it becomes `if (!options) {`. Same everywhere else. Thanks!

Reply:
My personal preference is `if(!options) {`, a local eslint config file would help to keep the same code design. 
Eslint will automatical fix design mistakes on my setup, so I had to disable eslint autofix for my PR.